lib/perl/Genome/Model/Tools/Transcriptome/ErccMapUnaligned.pm

```diff
@@ -9,24 +9,27 @@ use Path::Class qw();
 class Genome::Model::Tools::Transcriptome::ErccMapUnaligned {
     is => 'Command::V2',
     has_input => [
+        model => {
```

Comment:
Would it make sense for this tool to be able to accept a specific `Build` instead of always deriving one from the `Model`.  (This will be necessary if this tool is to be incorporated into the pipeline.)